packages/cli/src/models/initializer/truffle/Truffle.ts

```diff
@@ -72,8 +72,11 @@ const Truffle = {
         .filter((name) => {
           const contract = FileSystem.parseJsonIfExists(`${buildDir}/${name}`);
           if (contract) {
+            let isLibrary = contract.ast ? contract.ast.nodes[1].contractKind === "library" : false;
             const projectDir = buildDir.replace('build/contracts', '');
-            return contract.bytecode.length > 2 && contract.sourcePath.indexOf(projectDir) === 0;
+            return (contract.bytecode.length > 2 &&
+                    contract.sourcePath.indexOf(projectDir) === 0) &&
+                    !isLibrary
```

Comment:
This addresses #760 - note that it will completely omit libraries from the return of this method, if that's an issue this could be done in a more granular fashion. All tests passed locally and I couldn't immediately see a problem with omitting the libraries.